ice.go

```diff
@@ -26,6 +26,25 @@ type ICECandidateAttribute struct {
 	Value string
 }
 
+// https://tools.ietf.org/id/draft-ietf-mmusic-ice-sip-sdp-14.html
+
+// ICEMediaParameters media-level ICE parameters
+type ICEMediaParameters struct {
+	UserFragment string
+	Password     string
+	Mismatch     bool
+	Pacing       uint16
```

Comment:
We probably need a way to express `nil` here. Is it possible that a value of `0` means something that no pacing being sent at all?